components/usage-api/typescript/src/usage/v1/sugar.ts

```diff
@@ -176,59 +181,63 @@ export class PromisifiedUsageServiceClient {
         }
     }
 
-    public async getCostCenter(
-        attributionID: AttributionId,
-    ): Promise<CostCenter | undefined> {
+    public async getCostCenter(attributionID: AttributionId): Promise<CostCenter | undefined> {
         const request = new GetCostCenterRequest();
-        request.setAttributionId(AttributionId.render(attributionID))
+        request.setAttributionId(AttributionId.render(attributionID));
 
         const response = await new Promise<GetCostCenterResponse>((resolve, reject) => {
-            this.client.getCostCenter(
-                request,
-                (err: grpc.ServiceError | null, response: GetCostCenterResponse) => {
-                    if (err) {
-                        reject(err);
-                        return;
-                    }
-                    resolve(response);
-                },
-            );
+            this.client.getCostCenter(request, (err: grpc.ServiceError | null, response: GetCostCenterResponse) => {
+                if (err) {
+                    reject(err);
+                    return;
+                }
+                resolve(response);
+            });
         });
         if (!response.hasCostCenter()) {
             return undefined;
         }
-        return <CostCenter> {
-            id: AttributionId.parse(response.getCostCenter()!.getAttributionId()),
+
+        const attrId = AttributionId.parse(response.getCostCenter()!.getAttributionId());
```

Comment:
Nit: It would be a contract violation of the GRPC service to return a costCenter without an attributionID. We should enforce the contract in the API impl itself rather than on the TypeScript bindings.